src/components/Homepage/LandingPageWrapper.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { List } from "./List";
+import { Button } from "./Button";
+import { Description } from "./Description";
+import { Logo } from "./Logo";
+import { DescriptionWrapper } from "./DescriptionWrapper";
+import { BackgroundImage } from "./BackgroundImage";
+import background from "../../static/svg/backgroundImage.svg";
+
+export const LandingPageWrapper = () => {
+  return (
+    <BackgroundImage url={background} isCoverBackground={true}>
```

Comment:
there is shorhand for this "isCoverBackground"

Reply:
It has to be like this otherwise typescript gets an error